mlflow/server/js/src/experiment-tracking/components/CompareRunArtifactView.tsx

```diff
@@ -0,0 +1,69 @@
+import { useState } from 'react';
+import ShowArtifactPage from './artifact-view-components/ShowArtifactPage';
+import { RunInfoEntity } from '../types';
+import { useRunsArtifacts } from './experiment-page/hooks/useRunsArtifacts';
+import { getCommonArtifacts } from './experiment-page/utils/getCommonArtifacts';
+import { CompareRunArtifactViewSidebar } from './CompareRunArtifactViewSidebar';
+import { useDesignSystemTheme } from '@databricks/design-system';
+import './CompareRunArtifactView.css';
+import { FormattedMessage } from 'react-intl';
+
+export const CompareRunArtifactView = ({
+  runUuids,
+  runInfos,
+  colWidth,
+}: {
+  runUuids: string[];
+  runInfos: RunInfoEntity[];
+  colWidth: number;
+}) => {
+  const { theme } = useDesignSystemTheme();
+  const [artifactPath, setArtifactPath] = useState<string>();
```

Comment:
for more accurate typing

```suggestion
  const [artifactPath, setArtifactPath] = useState<string | undefined>();
```